src/components/Accordion/Accordion.scss

```diff
@@ -19,6 +19,10 @@
     font-weight: font-weight(bold);
     padding: 10px 12px;
     position: relative;
+    margin:0px;
+    display: flex;
+    justify-content: left;
+    color: #212121;
```

Comment:
@Farhan-emgage this should use the Heading or DisplayText component and not require custom styling.